packages/itwinui-react/src/core/ButtonGroup/ButtonGroup.tsx

```diff
@@ -114,6 +114,10 @@ export const ButtonGroup = React.forwardRef((props, ref) => {
           {...rest}
         >
           {(() => {
+            if (!overflowButton) {
+              return children;
```

Comment:
Should this be `return items;`?

Reply:
`children` is more direct and I think React has optimizations for directly passing through `children`. `items` is just an arrayed version of `children`.

Reply:
So then, should the second return ([code](https://github.com/iTwin/iTwinUI/pull/1939/files#diff-ced50949d49dd8fda606519a8fa8e193e7e7254479750d9b5c3dd56bbb354730R122)) be returning `children`?

Reply:
That code is not related to this PR. Can we leave it be?

I think of it as two main cases:
- non-overflow mode: always return the children untampered
- - overflow mode: always return an array (even if it may sometimes be _all_ children elements)